core/src/main/resources/defaults/clustertemplates/7.2.15/azure/datamart.json

```diff
@@ -36,7 +36,7 @@
         "template": {
           "attachedVolumes": [
             {
-              "count": 1,
+              "count": 0,
```

Comment:
Why change the volume count to 0 instead of changing the volume type to ephemeral? (assuming the required result is to use ephemeral volume only for this instance group)

Reply:
We were not able to create a datamart cluster in azure using ephemeral volumes only. So, we changed the volume count to 0. In this case, the ephemeral volume from the instance would be used which is of comparable size of attached volumes.

Reply:
Oh sorry, my mistake, I thought that this was az AWS template.